packages/catalog-model/src/entity/policies/FieldFormatEntityPolicy.ts

```diff
@@ -134,6 +134,12 @@ export class FieldFormatEntityPolicy implements EntityPolicy {
       require(`tags.${i}`, tags[i], this.validators.isValidTag);
     }
 
+    const links = entity.metadata.links ?? [];
+
+    for (let i = 0; i < links.length; ++i) {
+      optional(`links.${i}`, links[i]?.icon, this.validators.isValidEntityName);
```

Comment:
Wasn't sure if I should introduce a new check or re-use the entity name validation for the `icon` key. I also wasn't sure if I should add new `required` validation check for the `url`.

Reply:
Yeah there should for sure be a required check for that, but that's supposed to be covered by the `SchemaValidEntityPolicy`. So this new field should be added here with the proper semantics -> https://github.com/backstage/backstage/blob/master/packages/catalog-model/src/entity/policies/SchemaValidEntityPolicy.ts#L21

Reply:
Added additional validations and tests :)